### PR TITLE
multi proxy config

### PR DESCRIPTION
**Pull request by lzy2014love**
## PR Type

What kind of change does this PR introduce?

```html
[ ] Bugfix
[x] Feature
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #136

## Does this PR introduce a breaking change?

```text
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by lzy2014love** [comment]
Because when baseUri and proxyUri are arrays greater than 2, I'm not sure if I want to create multiple httpProxyServer

**Comment by rjoydip** [comment]
Any updates?

**Comment by rjoydip** [comment]
Okay. Let me take a look `httpProxyServer` module.

**Comment by rjoydip** [comment]
@lzy2014love Any updated?

File: src/Helper.ts [hunk 1]
```diff
@@ -141,9 +142,29 @@ export class Helper {
...
+            if (typeof baseUri === 'string' && typeof proxyUri === 'string') {
+                proxy = [[]];
+                proxy[0].push(baseUri, proxyUri);
+            } else if (baseUri instanceof Array && proxyUri instanceof Array) {
+                if (baseUri.length !== proxyUri.length) {
+                    console.error('When both baseUri and proxyUri are array, their length must be the same');
+                    window.showErrorMessage('When both baseUri and proxyUri are array, their length must be the same');
+                } else {
+                    proxy = [];
+                    for (let index = 0, length = baseUri.length; index < length; index++) {
```
[the remark this answers is not in the document]
Comment (lzy2014love) [reply src/Helper.ts]: Thank you very much.I will submit updated in the coming days

**Comment by rjoydip** [comment]
@lzy2014love I have checked the `HTTP proxy server` module there are no options for multiple proxy.

**Pull request closed without merging**